fdbkubernetesmonitor/README.md

```diff
@@ -0,0 +1,29 @@
+
+This package provides a launcher program for running FoundationDB in Kubernetes.
+
+To test this, run the following commands from the root of the FoundationDB
+repository:
+
+		docker build -t foundationdb/foundationdb-kubernetes:latest --build-arg FDB_VERSION=6.3.13 --build-arg FDB_LIBRARY_VERSIONS="6.3.13 6.2.30 6.1.13" -f packaging/docker/kubernetes/Dockerfile .
+		docker build -t foundationdb/foundationdb-kubernetes:latest-sidecar --build-arg FDB_VERSION=6.3.15 --build-arg FDB_LIBRARY_VERSIONS="6.3.15 6.2.30 6.1.13" -f packaging/docker/kubernetes/Dockerfile .
```

Comment:
Yes, it's intentional. The `latest` tag confuses things here. The goal of having two versions is to test upgrades. I'll change the tags to try to clarify that.

The `FDB_VERSION` argument is doing two things here. It allows building the images from previously uploaded binaries, and it is written into a file that allows the launcher to know what version of FDB it was built with. I'm open to other mechanisms for that to make things easier in CI, since that's the most common way the image is being built.